sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -215,7 +208,9 @@ function processFormData(formData?: FormDataMap) {
   for (const element in formData) {
     const item = formData[element];
     if (item instanceof Uint8Array) {
-      processedFormData[element] = binaryArrayToString(item);
+      // Some RLCs take a Uint8Array for the parameter, whereas FormDataMap expects
+      // a File or a Blob, so we need to wrap it.
+      processedFormData[element] = new Blob([item]);
```

Comment:
Will we set the default content type to `application/octet-stream` for binary data?

https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/payload/multipart/main.tsp#L31

Reply:
Good catch. We should be setting to `application/octet-stream` if there is no content type and it's binary data, but we are not doing that at the moment. I think this should be addressed upstream in `core-rest-pipeline`.

Reply:
Could you file an issue for this? We could talk to more details then.

Reply:
Opened #28081